pkgs/development/python-modules/quadprog/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "quadprog";
+  version = "0.1.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-jjIPn38UZhQeMX5hc26BCHIOatuuU/nWlg9ybv73iNk=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+  ];
+
```

Comment:
The upstream repo contains [tests](https://github.com/quadprog/quadprog/tree/master/tests). If possible, the tests should be executed.

Please add `pythonImportsCheck`.

Reply:
The tests don't seem to be present on PyPI. Switched to fetchFromGitHub for now.